### PR TITLE
fix(contract): linting

### LINKED ISSUE #19: linting issues when running `yarn lint`
When running `yarn lint` on the projet directory I am getting the following errors:

```
dylan@dylgal-linlt:~/Desktop/demo-new-docs$ yarn lint
yarn run v1.22.21
$ yarn workspaces run lint

> demo-new-docs-contract
$ eslint '**/*.{js,jsx}'

Oops! Something went wrong! :(

ESLint: 8.53.0

ESLint couldn't find the config "@agoric" to extend from. Please check that the name of the config is correct.

The config "@agoric" was referenced from the config file in "/home/dylan/Desktop/demo-new-docs/contract/package.json".

If you still have problems, please stop by https://eslint.org/chat/help to chat with the team.

error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed.
Exit code: 2
Command: /home/dylan/.nvm/versions/node/v18.16.0/bin/node
Arguments: /home/dylan/.yarn/releases/yarn-1.22.21.cjs run lint
Directory: /home/dylan/Desktop/demo-new-docs/contract
Output:

info Visit https://yarnpkg.com/en/docs/cli/workspaces for documentation about this command.
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

```

### PR DESCRIPTION
**Pull request by 0xpatrickdev**
- closes #19
- keeping scope small, and will include `@typescript-eslint` configuration in a separate PR

### CODE REVIEW

File: .github/workflows/pr.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+name: Lint and Test
+
+on:
+  pull_request:
```
Comment (dckc) [inline .github/workflows/pr.yml:4]: :clap:

File: .github/workflows/pr.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+jobs:
+  all:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+      - name: Use Node.js 18.8.x
```
Comment (dckc) [inline .github/workflows/pr.yml:12]: here's hoping we can do both major LTS versions
(maybe add a comment about why not yet? https://github.com/Agoric/agoric-sdk/issues/8636 )

**Review by dckc — APPROVED** [review]
nice!

File: contract/package.json [hunk 3]
```diff
@@ -13,10 +13,8 @@
     "start": "yarn docker:make clean start-contract print-key",
     "build": "exit 0",
     "test": "ava --verbose",
-    "lint": "eslint '**/*.{js,jsx}'",
-    "lint-fix": "eslint --fix '**/*.{js,jsx}'",
-    "lint-fix-jessie": "eslint -c '.eslintrc-jessie.js' --fix '**/*.{js,jsx}'",
-    "lint-jessie": "eslint -c '.eslintrc-jessie.js' '**/*.{js,jsx}'"
+    "lint": "eslint '**/*.js'",
+    "lint:fix": "eslint --fix '**/*.js'"
```
Comment (dckc) [inline contract/package.json:17]: :+1:

File: .github/workflows/pr.yml (hunk 2 above)
Comment (0xpatrickdev) [reply .github/workflows/pr.yml:12]: 👍 [fccae72](https://github.com/Agoric/dapp-offer-up/pull/33/commits/fccae72fed0f1a6c5286bd6f587f1b50e4a77480)

**Pull request merged**